src/rgw/rgw_file.h

```diff
@@ -740,7 +740,7 @@ namespace rgw {
   class RGWLibFS
   {
     CephContext* cct;
-    struct rgw_fs fs;
```

Comment:
this example shows that the {} style is inconsistent;  would it be more consistent to use ctor-initializer syntax?